### PR TITLE
Added height, width, button and company information

### PR DESCRIPTION
**Pull request by yormanegc**
Bugs were fixed, missing to complete the task. Within it, the height, width, company information and the button (without functionality) were placed.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 // Dependencies
 import { Routes, Route } from 'react-router-dom'
+import Navbar from './components/Navbar'
 
 function App() {
   return (
     <div className="App">
+      <Navbar />
```
Comment (karttofer) [inline src/App.js]: Delete this

File: src/assets/style/components/navbar.scss [hunk 2]
```diff
@@ -7,29 +7,78 @@
...
+
 //Navbar
 .navbar {
   @include navbarDefaultGroup;
 
-  &__left {
-    &-img-responsive {
-      @include funcionality(none);
+  &__container {
+    @include container();
```
Comment (karttofer) [inline src/assets/style/components/navbar.scss]: You don't need the () in the container keyword

**Review by karttofer — COMMENTED** [review]
Please review this

File: src/assets/style/components/navbar.scss (hunk 2 above)
Comment (yormanegc) [reply src/assets/style/components/navbar.scss]: I'm sorry 'the prettiest' adds it I don't know why

File: src/App.js (hunk 1 above)
Comment (yormanegc) [reply src/App.js]: Go

**Pull request merged**